modules/logging/pom.xml

```diff
@@ -24,6 +24,11 @@
       <artifactId>jcl-over-slf4j</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.owasp</groupId>
+      <artifactId>security-logging-logback</artifactId>
+      <version>1.1.3</version>
```

Comment:
move versions to our BOM: https://github.com/oasp/oasp4j/blob/develop/bom/pom.xml